test/parser/parse.test.ts

```diff
@@ -84,6 +84,58 @@ describe('parse', () => {
         expect(Boolean(out.flags.myflag2)).to.equal(true)
       })
 
+      it('throws error when no value provided to required flag', async () => {
+        try {
+          await parse(['--myflag', '--second', 'value'], {
+            flags: {myflag: Flags.string({required: true}), second: Flags.string()},
+          })
```

Comment:
The tests need to throw if `parse()` doesn't throw an error, otherwise it could cause a false positive.